wp-cli-templates/block-register-php.mustache

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
When committing PHP file generated by the cli script (`yarn docker:wp jetpack block hello`), I get:

```
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
  3 | ERROR | Doc comment short description must start with a capital
    |       | letter
    |       | (Generic.Commenting.DocComment.ShortNotCapital)
 16 | ERROR | Doc comment short description must start with a capital
    |       | letter
    |       | (Generic.Commenting.DocComment.ShortNotCapital)
----------------------------------------------------------------------
```

Reply:
Is this something wrong with this branch? The phpdoc produced is valid. I think it's more of an issue with PHP code sniffer

https://github.com/squizlabs/PHP_CodeSniffer/issues/377

Reply:
I think it appears if you give lowercase block name like I did. If I give a block name with capital first letter, no problemos.

Maybe the script could `ucfirst` the title CLI argument to be safe?